addons/project/views/project_views.xml

```diff
@@ -567,75 +571,84 @@
+                                        <div class="o_setting_left_pane">
+                                            <field name="allow_task_dependencies"/>
+                                        </div>
+                                        <div class="o_setting_right_pane">
+                                            <label for="allow_task_dependencies"/>
+                                            <div class="text-muted">
+                                                Determine the order in which to perform tasks
+                                            </div>
+                                        </div>
                                     </div>
-                                    <div class="o_setting_right_pane">
-                                        <label for="allow_task_dependencies"/>
-                                        <div class="text-muted">
-                                            Determine the order in which to perform tasks
+                                    <div class="o_setting_box mt-4" id="project_milestone_setting" groups="project.group_project_milestone">
+                                        <div class="o_setting_left_pane">
+                                            <field name="allow_milestones"/>
+                                        </div>
+                                        <div class="o_setting_right_pane">
+                                            <label for="allow_milestones"/>
+                                            <div class="text-muted">
+                                                Track major progress points that must be reached to achieve success
+                                            </div>
                                         </div>
                                     </div>
                                 </div>
-                                <div class="col-lg-6 o_setting_box" id="project_milestone_setting" groups="project.group_project_milestone">
-                                    <div class="o_setting_left_pane">
-                                        <field name="allow_milestones"/>
-                                    </div>
-                                    <div class="o_setting_right_pane">
-                                        <label for="allow_milestones"/>
-                                        <div class="text-muted">
-                                            Track major progress points that must be reached to achieve success
+                            </group>
+                            <group name="group_time_managment" string="Time Managment" invisible='1' class="row mt16 o_settings_container col-lg-6">
+                                <div/>
+                            </group>
+                            <group name="group_documents_analytics" string="Documents &amp; Analytics" class="row mt16 o_settings_container col-lg-6" attrs="{'invisible': [('rating_active', '=', False)]}">
```

Comment:
In the Project res.settings, the "Documents & Analytics" section is just called "Analytics" when the Documents app is not installed.
Maybe we could do the same here?

Reply:
@MissingNoShiny , Done all the changes. thanks